R/logging.R

```diff
@@ -102,6 +104,7 @@ logger_detailed <- function(logger, level, req, caller, ...) {
     caller = caller,
     method = req$REQUEST_METHOD,
     path = req$PATH_INFO,
+    endpoint = req$endpoint,
```

Comment:
I couldn't find a way to get this already from what is available in the `porcelain_log_postserialize` but let me know if there is another way to get it

Reply:
yeah, I have no idea; your approach seems sensible